src/patternfly/demos/Page/examples/page-component-expandable-nav-example.hbs

```diff
@@ -1,6 +1,3 @@
-{{#> background-image}}
```

Comment:
should you also remove background-image from the data-list examples

Reply:
yup, also from datatable.